docs/src/man/spaces.md

```diff
@@ -169,6 +169,12 @@ generally, the simple objects of a fusion category.
     complex data, a one-time warning will be printed, but most operations should continue
     to work nonetheless.
 
+One more important instance of `ElementarySpace` is the `GradedSpace`, which is used to
+represent a graded complex vector space with Euclidean product, where the grading is
```

Comment:
```suggestion
represent a graded complex vector space with Euclidean inner product, where the grading is
```

There is no such thing as a Euclidean product, only Euclidean inner product! :grin: